packages/arb-token-bridge-ui/src/components/App/AppContext.tsx

```diff
@@ -28,6 +45,8 @@ type Action =
   | { type: 'layout.set_is_transferring'; payload: boolean }
   | { type: 'layout.set_txhistory_panel_visible'; payload: boolean }
   | { type: 'layout.set_txhistory_show_sent_tx'; payload: boolean }
+  | { type: 'layout.set_txhistory_show_cctp_deposits'; payload: boolean }
+  | { type: 'layout.set_txhistory_default_tab'; payload: TransactionHistoryTab }
```

Comment:
Not sure what `default` means here. I'd just rename it to `set_txhistory_tab`

Reply:
Make sense, I removed it